iaac/terraform/kubeflow-components/admission-webhook/versions.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_version = ">= 1.2.7"
+
+  required_providers {
+    aws = {
```

Comment:
this needs aws provider? same for others

Reply:
broke without it, probably because helm-addon module has an optional irsa module resource (interacts with aws)